### ISSUE TITLE
UX issue

### ISSUE THREAD

**Issue by sleepylee** [rahatshahzeb/N3#10]
try refreshing several times, say I'm at item #3, if there are no changes why would I be delivered back to the first list item?

**Comment by rahatshahzeb** [comment]
Actually its not refreshing several times. I implemented cache to store data to show previously loaded data whenever user visits the app again. Since the app also requests for new data from server on start, which in return repopulate the list, it gives an impression like the refresh call is hit multiple times. 

We can handle this if we maintain the current scrolled position and move the recyclerview to that position whenever new data is fetched.

**Comment by sleepylee** [comment]
I agree with you the test didn't requires the db caching, but as natural sense, if we deliver something, shouldn't it be perfect? Otherwise I believe the focus should had happened on the architecture of this simple app. However the cache feature is still a good point.
